src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -420,6 +442,12 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
     case Event(message: CallStarted, _) =>
       resendDueToPendingExecutionWrites(message)
       stay()
+    case Event(message @ CallCompleted(finalCallKey: FinalCallKey, _, _, _, _, _), data) if !data.isPersistedRunning(finalCallKey) =>
+      // Unlike "real" calls which are managed by a CallActor, the final call completion message is triggered
+      // internally and there isn't an automatic retry if the message isn't processed.  So if the message can't be
+      // processed due to pending writes, schedule it to be resent here.
+      resendDueToPendingExecutionWrites(message)
```

Comment:
This seems like a good idea, along with farming out the copy work as a real backend call.  Per in-person discussion this would wait until one of the forthcoming tickets that will ask for additional things to be copied.